openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -959,6 +962,22 @@ Cache-Control: no-store
 }
 ```
 
+Below is a non-normative example of a Credential Response in an immediate issuance flow for multiple Credential instances in JWT VC format (JSON encoded):
+
+```
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "credentials": [
```

Comment:
again an chance to be more specific with media types.

perhaps an example of CWT would be good here, can the response contain both JWT and CWT in a single response?

Reply:
I believe the argument here was that the Client/Wallet knows which Credential Configuration, i.e. Credential Format, is requested in the Credential Request, therefore this information would be redundant.

Reply:
please open a separate issue for this, if you want to pursue this, I believe it is out of scope for this PR